scripts/bootloader/app_signed.hex

```diff
@@ -0,0 +1,13794 @@
+:020000040000FA
```

Comment:
I assume this file is not intentionally included?

Reply:
Yes, I'll remove it.